### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EduSe7en42**
Editing README.md for better understanding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,73 @@
...
+```
+TWITTER_TWEETJS_FILEPATH=tweets.js
+```
+#### Linux and MacOS
+1. Same thing as Windows;
+2. Open Terminal;
+3. Got to your Desktop folder;
+4. Write
+```
+export MASTODON_API_BASEPATH=https://[YOUR-MASTODON-INSTANCE]
```
Comment (brandonhaslegs) [inline README.md:47]: What exactly is my instance? My Mastodon username?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,73 @@
...
-
-#### Clone this repo and run
+### #3 - Clone this repo
+Clone this repo in your Desktop folder with the command: 
+```
+git clone https://github.com/FGRibreau/import-tweets-to-mastodon.git
+```
+Or download the zip and extract in Desktop.
+### #4 - Set an API key from Mastodon
+To request an API key from Mastodon, click [here](https://mastodon.cloud/settings/applications). Create an new application in "New Application". Set an Application Name (e. g.: api-mastodon-twitter) and, at the end of page, click in "Submit".
```
Comment (brandonhaslegs) [inline README.md:25]:
```suggestion
To request an API key from Mastodon, click [here](https://mastodon.cloud/settings/applications). Create a new application with "New Application". Set an Application Name (e. g.: api-mastodon-twitter) and, at the end of page, click "Submit". 
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,73 @@
...
+Clone this repo in your Desktop folder with the command: 
+```
+git clone https://github.com/FGRibreau/import-tweets-to-mastodon.git
+```
+Or download the zip and extract in Desktop.
+### #4 - Set an API key from Mastodon
+To request an API key from Mastodon, click [here](https://mastodon.cloud/settings/applications). Create an new application in "New Application". Set an Application Name (e. g.: api-mastodon-twitter) and, at the end of page, click in "Submit". 
+### #5 - Set the environment variables
+#### Windows
+1. Copy your "Your access token" from developer's page in Mastodon;
```
Comment (brandonhaslegs) [inline README.md:28]:
```suggestion
1. Copy your "Your access token" from the Application page you just created.
```

File: README.md [hunk 4]
```diff
@@ -1,26 +1,73 @@
...
+MASTODON_API_BASEPATH=https://[YOUR-MASTODON-INSTANCE] 
+```
+```
+MASTODON_API_KEY=YOUR_ACCESS_TOKEN_FROM_MASTODON
+```
+```
+TWITTER_TWEETJS_FILEPATH=tweets.js
+```
+#### Linux and MacOS
+1. Same thing as Windows;
```
Comment (brandonhaslegs) [inline README.md:42]:
```suggestion
1. Copy your "Your access token" from the Application page you just created.
```

File: README.md [hunk 5]
```diff
@@ -1,26 +1,73 @@
...
+2. Open Terminal;
```
Comment (brandonhaslegs) [inline README.md:43]:
```suggestion
2. Open Terminal.
```

File: README.md [hunk 6]
```diff
@@ -1,26 +1,73 @@
...
+3. Got to your Desktop folder;
```
Comment (brandonhaslegs) [inline README.md:44]:
```suggestion
3. Go to your Desktop folder (or the folder you installed it in).
```

File: README.md [hunk 7]
```diff
@@ -1,26 +1,73 @@
...
+4. Write
```
Comment (brandonhaslegs) [inline README.md:45]:
```suggestion
4. Run these commands:
```

File: README.md [hunk 8]
```diff
@@ -1,26 +1,73 @@
...
+```
+```
+export MASTODON_API_KEY=YOUR_ACCESS_TOKEN_FROM_MASTODON
+```
+```
+export TWITTER_TWEETJS_FILEPATH=../tweets.js
+```
 
+### #6 - Setting the script
+In the terminal (or CMD), enter in "import-tweets-mastodon" folder, located in Desktop. Write
```
Comment (brandonhaslegs) [inline README.md:57]:
```suggestion
In the terminal (or CMD), navigate to the "import-tweets-mastodon" folder, located in Desktop (or wherever you installed it). Run these commands: 
```

File: README.md [hunk 9]
```diff
@@ -1,26 +1,73 @@
...
 
+### #6 - Setting the script
+In the terminal (or CMD), enter in "import-tweets-mastodon" folder, located in Desktop. Write 
 ```
-git clone git@github.com:FGRibreau/import-tweets-to-mastodon.git
-cd import-tweets-to-mastodon
 npm install
-MASTODON_API_BASEPATH=https://mastodon-instance.com MASTODON_API_KEY=YOUR_TOKEN TWITTER_TWEETJS_FILEPATH=/path/to/tweet.js node import.js
+```
+After this
```
Comment (brandonhaslegs) [inline README.md:62]:
```suggestion
Then, run this:
```

File: README.md [hunk 10]
```diff
@@ -1,26 +1,73 @@
...
-cd import-tweets-to-mastodon
 npm install
-MASTODON_API_BASEPATH=https://mastodon-instance.com MASTODON_API_KEY=YOUR_TOKEN TWITTER_TWEETJS_FILEPATH=/path/to/tweet.js node import.js
+```
+After this 
+```
+node import.js
 ```
 
 Tips: add the `DEBUG=*` environment variable for verbose output.
```
Comment (brandonhaslegs) [inline README.md]:
```suggestion
Tip: add the `DEBUG=*` environment variable for verbose output.
```

File: README.md [hunk 11]
```diff
@@ -1,26 +1,73 @@
...
 
-:tada:
-
 #### Need help?
 
-- This project is open-sourced as is. 
-- I won't do support for free.
+- Send an email to eduardopribeiro.6899@tutanota.com;
```
Comment (brandonhaslegs) [inline README.md:70]:
```suggestion
- Send an email to eduardopribeiro.6899@tutanota.com.
```

File: README.md [hunk 12]
```diff
@@ -1,26 +1,73 @@
...
+- I'll not have all the answers, but I'll try to help you;
```
Comment (brandonhaslegs) [inline README.md:71]:
```suggestion
- I won't have all the answers, but I'll try to help you.
```

File: README.md [hunk 13]
```diff
@@ -1,26 +1,73 @@
...
+- This project is open-sourced. Feel free to contribute;
```
Comment (brandonhaslegs) [inline README.md:72]:
```suggestion
- This project is open-sourced. Feel free to contribute.
```

File: README.md [hunk 14]
```diff
@@ -1,26 +1,73 @@
-# How to imports Tweets into Mastodon (e.g. https://framapiaf.org )
+# Importing tweets for a Mastodon instance (e. g.: https://framapiaf.org) :bird::arrow_right::elephant:
+Thats a simples script to import yours tweets from any Twitter account to a Mastodon account, in any instance.
```
Comment (brandonhaslegs) [inline README.md:2]:
```suggestion
This is a simple script to import yours tweets from any Twitter account to a Mastodon account, in any instance.
```

File: README.md [hunk 15]
```diff
@@ -1,26 +1,73 @@
-# How to imports Tweets into Mastodon (e.g. https://framapiaf.org )
+# Importing tweets for a Mastodon instance (e. g.: https://framapiaf.org) :bird::arrow_right::elephant:
```
Comment (brandonhaslegs) [inline README.md:1]:
```suggestion
# Importing tweets to a Mastodon instance (e.g.: https://framapiaf.org) :bird::arrow_right::elephant:
```

File: README.md [hunk 16]
```diff
@@ -1,26 +1,73 @@
...
+Thats a simples script to import yours tweets from any Twitter account to a Mastodon account, in any instance.
 
-#### Ask twitter for a data export
+## Steps
+For this tutorial, you'll need to install **Node.js** and **Git**.
+ - Node.js: [Download | Node.js (nodejs.org)](https://nodejs.org/en/download/)
+ - Git: [Git - Downloads (git-scm.com)](https://git-scm.com/downloads)
+### #1 - Request your Twitter data in link below;
```
Comment (brandonhaslegs) [inline README.md:8]:
```suggestion
### #1 - Request your Twitter data with the link below:
```

File: README.md [hunk 17]
```diff
@@ -1,26 +1,73 @@
...
+Go [here](https://twitter.com/settings/your_twitter_data) for request your Twitter data. You'll recive a link to download a zip file. There will be tweets, images, information etc. It'll be in this configuration:
```
Comment (brandonhaslegs) [inline README.md:9]:
```suggestion
Go [here](https://twitter.com/settings/your_twitter_data) to request your Twitter data. You'll receive a link to download a ZIP file. There will be tweets, images, information etc. It'll be in this configuration:
```

File: README.md [hunk 18]
```diff
@@ -1,26 +1,73 @@
...
+ - twitter
+    - assets (folder)
+    - data (folder) 
+    - Your archive (html file)
 
-Go [here](https://twitter.com/settings/your_twitter_data).
+### #2 - Extract archive and look for `tweets.js`
+Extract the file in your Desktop folder. Will be more easy for manipulate. Find the **tweets.js** file. It'll be, probably inside "data" folder and move it to Desktop.
```
Comment (brandonhaslegs) [inline README.md:16]:
```suggestion
Extract the file to your Desktop folder (or any folder that works for you). Find the **tweets.js** file. It'll probably be inside the "data" folder. Move it to your Desktop.
```

File: README.md (hunk 1 above)
Comment (brandonhaslegs) [inline README.md:47]:
```suggestion
export MASTODON_API_BASEPATH=https://[YOUR-MASTODON-INSTANCE]

for example `export MASTODON_API_BASEPATH=https://https://mastodon.social/@brandonhaslegs`
```

Comment (EduSe7en42) [reply README.md:47]: Your instance on Mastodon (eg "https://mastodon.cloud", "https://mstdn.io")

**Comment by FGRibreau** [comment]
please resolve conflicts so I can  merge it :)

**Comment by EduSe7en42** [comment]
> please resolve conflicts so I can merge it :)

Done ✅

**Comment by FGRibreau** [comment]
merged, thanks!

**Pull request merged**